corehq/sql_accessors/migrations/0012_soft_deletion.py

```diff
@@ -16,7 +13,6 @@ class Migration(migrations.Migration):
 
     operations = [
         migrator.get_migration('soft_delete_cases.sql'),
-        migrator.get_migration('get_form_ids_for_user.sql'),
```

Comment:
Out of curiosity, why is this being removed? It might be better to add a new migration operation that removes it rather than modifying an old migration. The same applies to other migrations that are being modified in a way that affects database state.

Reply:
This is standard practice for updating the SQL functions so that we can remove the file that contains the old function in the codebase. There is a migration added in this PR which removes the function itself from the DB:
* https://github.com/dimagi/commcare-hq/pull/33510/files#diff-e3deef1ca541f7a2ba52c10544f0374ee069e1dd0deeaf9cca03eb4a9df2ec03
* https://github.com/dimagi/commcare-hq/pull/33510/files#diff-7e7b60e23f78e59e213c2ad346a67b6c66779c4b7555c3e99f9e66b8014e603f


(See step 3 at the bottom of [this section](https://github.com/dimagi/commcare-hq/tree/master/corehq/form_processor#updating-a-sql-function))